src/components/dialog/dialog.svelte

```diff
@@ -141,8 +136,8 @@
     border: none;
     display: grid;
 
-    width: var(--leo-dialog-width, 500px);
-    max-width: calc(100% - var(--leo-spacing-m) * 2);
+    width: calc(100% - var(--leo-spacing-m) * 2);
```

Comment:
does it work without this calc? I feel like it should because the padding is applied inside the dialog element (i.e. this will result in a `4xl` padding around the dialog, rather than a 2xl)

Reply:
The `calc` isn't for the padding, but rather the margin between the viewport and the dialog.

| Without `calc` | With `calc` |
| -- | -- |
| ![SCR-20241206-senr](https://github.com/user-attachments/assets/0b5f3c10-d523-4c57-8c86-cdffe24a5c32) | ![SCR-20241206-sfbj](https://github.com/user-attachments/assets/af3f604c-872e-4dda-a88c-7187174f366f) |